build/golang/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM golang:1.8.0-stretch
+RUN apt-get update && apt-get install -y python-requests python-yaml file jq unzip protobuf-compiler libprotobuf-dev && \
```

Comment:
Looks like gocertifi comes with a python task which uses these, which we use in the service build (and probably others, I assume)